website/content/docs/runner/index.mdx

```diff
@@ -7,40 +7,56 @@ description: |-
 
 # Waypoint Runners
 
-Waypoint utilizes "runners" to execute operations. Every build, deploy, or
-release operation happens on a Waypoint runner. Waypoint has three kinds of
-runners:
+Waypoint utilizes "runners" to execute operations. A runner is a program executing
+[`waypoint runner agent`](/command/runner-agent) that is configured with the details
+of the Waypoint server.
 
-### CLI Runner
+Every operation that Waypoint runs (build, deploy, release, etc) is executed by a Waypoint runner.
+The Waypoint server maintains a queue of jobs that the server hands out to connected
+runners. The server never connects directly to runners, runners always connect to the
+server. This is important because runners are designed to be run in the same
+environment that your application is deployed into because the plugins such as Deploy
+need to talk directly to your deployment platform APIs.
+
+In most deployments, there are two kinds of runners:
 
-When you trigger an action with the CLI, i.e. by running `waypoint deploy`, the CLI acts as a runner
-capable of performing your operation. If your deployment needs AWS credentials, or a valid kubectl context,
-it can access the credentials on the developer workstation that it's running on.
+### Static Runner
 
-If you only use Waypoint from the CLI, this is the only kind of runner that is used.
+Static runners are long-lived runners that are installed into your deployment
+environment once, likely when you setup Waypoint for the first time.
 
-### Remote Runner
+The [`waypoint install`](/commands/install) and  [`waypoint runner install`](/commands/install)
+commands can install these runners for you, using details about your deployment
+environment.
 
-Not every operation in Waypoint needs to happen in response to a CLI invocation
-from a privileged workstation. Waypoint can trigger deployments from the UI or
-automatically based on git changes, or automatically poll the status of
-applications in the background. In these cases, Waypoint needs a "remote
-runner", a long-lived runner that is always available to run these operations.
+One issue with static runners is that they have fixed capacity: they can only run so
+many operations at a time. Additionally to run certain plugins like Docker, need to have
```

Comment:
```suggestion
many operations at a time. Additionally to run certain plugins like Docker, we need to have
```

Reply:
Maybe instead of `we` I'd say `you would`